pages/k8s/1.16/components.md

```diff
@@ -0,0 +1,329 @@
+This allows TLS termination to happen on the workload. Refer to the
+[upstream documentation](https://kubernetes.github.io/ingress-nginx/user-guide/tls/#ssl-passthrough)
+for more information.
+
+### Improved LXD support
+
+LXD containers used for hosting Kubernetes components require some specific profile settings. These
+profiles are now embedded in the charms themselves and applied when deployed, dramatically
+simplifying the process of installing Charmed Kubernetes on a single machine. See the
+[Local install documentation](/kubernetes/docs/install-local) for the updated instructions.
+
+### Improved Prometheus/Grafana integration
+
+The setup and configuration of Prometheus and Grafana has been significantly streamlined with
+new relations to allow the charms to manage the scraper job and dashboards. This means that
+monitoring can now be added by specifying a single overlay when deploying Charmed Kubernetes.
+Refer to the [updated documentation](/kubernetes/docs/monitoring) for more information.
+
+### Improved OpenStack integration
+
+The OpenStack Integrator charm can now replace the Kube API Load Balancer by providing a
+native OpenStack load balancer (Octavia or Neutron) to provide HA load balancing for the
+Kubernetes control plane. Refer to the [updated documentation](/kubernetes/docs/openstack-integration)
+for more information.
+
+### Docker Registry with Containerd
+
+The Docker registry charm can now be related directly to the Containerd runtime charm.
+Refer to the [documentation](/kubernetes/docs/docker-registry) for instructions on how to deploy the charm.
+
+### Renamed default container registry
+
+The Canonical container image registry has a new, firewall-friendly name:
+`image-registry.canonical.com:5000` is now `rocks.canonical.com`. The old URL
+is an alias for `rocks` and will continue to work. However, the default
+configuration for current charms has changed to the new URL.
+
+## Fixes
+
+A list of bug fixes and other minor feature updates in this release can be found at
+[https://launchpad.net/charmed-kubernetes/+milestone/1.16](https://launchpad.net/charmed-kubernetes/+milestone/1.16).
+
+Special thanks to [pierrop](https://github.com/pierrop) for contributing a fix to
+[issue 1841965](https://bugs.launchpad.net/charm-kubernetes-master/+bug/1841965)!
+
+## Known Issues
+
+The Kubernetes Dashboard shipped with Charmed Kubernetes 1.16 is version 2.0.0-beta4. While unusual to ship a beta component with a stable release, in this case it was necessary, since the latest stable dashboard (v1.10.1) does not work with Kubernetes 1.16.
+
+
+## Core charms
+
+These charms are the core components or official optional components of the
+release. These charms are maintained by the Charmed Kubernetes team.
+
+<table class ="u-table-layout--auto">
+  <thead>
+    <tr>
+      <th>Charm</th>
+      <th>Summary</th>
+      <th>docs</th>
+      <th>source</th>
+      <th>bugs</th>
+      <th>version</th>
+      <th>notes</th>
```

Comment:
Thoughts on how we'll populate these columns? Will we autogen this page once, then edit it manually?

Reply:
The version can be autogen, the notes will have to be added manually, because I don't know where else we could source those